packages/material-ui-unstyled/src/SliderUnstyled/SliderUnstyled.js

```diff
@@ -144,44 +145,40 @@ function doesSupportTouchActionNone() {
   return cachedSupportsTouchActionNone;
 }
 
-const getUtilityClass = (name) => {
-  return `MuiSlider-${name}`;
-};
-
 const useSliderClasses = (props) => {
   const { color, disabled, marked, orientation, track, classes = {} } = props;
 
   const utilityClasses = {
     root: clsx(
-      getUtilityClass('root'),
+      sliderClasses['root'],
       classes['root'],
-      getUtilityClass(`color${capitalize(color)}`),
+      sliderClasses[`color${capitalize(color)}`],
```

Comment:
Will we get requests to support `<Slider color="error" />`? No idea.

Reply:
I see the concern, should we maybe use the dynamic method for dynamic props? And have in the sliderClasses the ones we support by default?

Reply:
On the other hand we could not do this changes, and leave as it was before with the `getUtilityClass` utility, and export hte classes just for the sake of the clients being able to use them, and use them also in the tests...

Reply:
I don't know. The propTypes are another bottlneck. I think that we can wait.

Reply:
Alright, let's leave it open again for visibility